src/main/java/org/terasology/gooeyDefence/EnemyManager.java

```diff
@@ -16,6 +16,7 @@
 
 package org.terasology.gooeyDefence;
 
+import org.joml.Quaternionf;
```

Comment:
Is this actually not used anymore? I was trying to test an unrelated logistics thing with dependencies and module sandboxing, this first failed to be found as expected, then I fixed and thing and it could be found, but then I noticed it isn't actually used anywhere?